README.md

```diff
@@ -202,17 +204,17 @@ There are two ways that arguments can be passed into the script. The first, is v
-| &#x2011;&#x2011;artifact            | The URI of the artifact. (`<source>://<organization>/<repository>/<artifact_id>`)                                                                                                                                                            | --commit   |
-| &#x2011;&#x2011;pull_request_number | The pull request number of the commit. (e.g. 123). Used only if --commit is included                                                                                                                                                         | --commit   |
+| Argument                            | Description                                                                                                                                | Dependency |
+| ----------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------ | ---------- |
+| &#x2011;&#x2011;run                 | The URI of the job run that built the code. (`<source>://<organization>/<pipeline>/<run_id>`)                                              |            |
+| &#x2011;&#x2011;run_status          | The status of the job run that built the code. (Allowed values: `Success`, `Failed`, `Canceled`, `Queued`, `Running`, `Unknown`, `Custom`) | --run      |
+| &#x2011;&#x2011;run_status_details  | Any extra details about the status of the job run.                                                                                         | --run      |
+| &#x2011;&#x2011;run_start_time      | The start time of the job run in milliseconds since the epoch, ISO-8601, or `Now`. (e.g. `1626804346019`, `2021-07-20T18:05:46.019Z`)      | --run      |
+| &#x2011;&#x2011;run_end_time        | The end time of the job run in milliseconds since the epoch, ISO-8601, or `Now`. (e.g. `1626804346019`, `2021-07-20T18:05:46.019Z`)        | --run      |
+| &#x2011;&#x2011;run_name            | The name of the job run that built the code.                                                                                               | --run      |
+| &#x2011;&#x2011;commit              | The URI of the commit. (`<commit_source>://<commit_organization>/<commit_repository>/<commit_sha>`)                                        | --run      |
+| &#x2011;&#x2011;artifact            | The URI of the artifact. (`<source>://<organization>/<repository>/<artifact_id>`)                                                          | --commit   |
+| &#x2011;&#x2011;pull_request_number | The pull request number of the commit. (e.g. 123). Used only if `--commit` is included                                                     | --commit   |
```

Comment:
We might not need to say: `Used only if `--commit` is included` as `--commit` is the dependency.